plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -402,6 +402,11 @@ export async function createRouter(
       delete task.secrets;
       res.status(200).json(task);
     })
+    .post('/v2/tasks/:taskId', async (req, res) => {
```

Comment:
I think we'll simply need to go against REST here and do something like `/v2/tasks/:taskId/abort`. I don't feel a `DELETE` here makes sense because that's not what's happening to the resources. I guess if we wanna be as REST as possible it would be something like `DELETE /v2/tasks/:taskId/current-task-run` but that just feels wrong too.

Reply:
Yeah, I checked other tools how they perform `abort`, and they used `post`. Like for example here: https://learn.microsoft.com/en-us/rest/api/playfab/admin/scheduled-task/abort-task-instance?view=playfab-rest.

Will change to `/v2/tasks/:taskId/abort`